eng/pipelines/runtime.yml

```diff
@@ -206,15 +206,19 @@ extends:
           - name: timeoutPerTestCollectionInMinutes
             value: 180
           jobParameters:
-            testGroup: innerloop
             timeoutInMinutes: 120
             nameSuffix: NativeAOT
             buildArgs: -s clr.aot+host.native+libs -rc $(_BuildConfig) -lc Release -hc Release
             extraStepsTemplate: /eng/pipelines/coreclr/nativeaot-post-build-steps.yml
             extraStepsParameters:
               creator: dotnet-bot
               testBuildArgs: nativeaot tree nativeaot
+              liveLibrariesBuildConfig: Release
```

Comment:
It looks like this code is passing `-rc $(_BuildConfig)` above. How does the `liveLibrariesBuildConfig` setting affect this?

Reply:
The `liveLibrariesBuildConfig` for the extra stepsneeds to match whatever is passed for the `-lc` parameter above if the libraries are built different than the runtime. The `nativeaot-post-build-steps.yml` template assumes that the passed in `$(_BuildConfig)` was used to build the runtime.